pkg/kubelet/util/nodelease.go

```diff
@@ -46,7 +46,7 @@ func SetNodeOwnerFunc(c clientset.Interface, nodeName string) func(lease *coordi
 					},
 				}
 			} else {
-				klog.Errorf("failed to get node %q when trying to set owner ref to the node lease: %v", nodeName, err)
+				klog.ErrorS(err, "Failed to get node when trying to set owner ref to the node lease", "node", nodeName)
```

Comment:
```suggestion
				klog.ErrorS(err, "Failed to get node when trying to set owner ref to the node lease", "nodeName", nodeName)
```

Reply:
I think node is fine?

Reply:
looks fine to me too, looking at `node` key [here](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#use-klogkobj-and-klogkref-for-kubernetes-objects)

excerpts:
For non-namespaced object we can pass empty string to namespace argument
```
func updateNode(nodeName string) {
   ...
   klog.InfoS("Updated node", "node", klog.KRef("", nodeName))
}
```

can change to `KRef` way but the result would be same, wdyt ?

Reply:
I think this is a fine MVP and we can leave this as is for now, this is in beta after all. We'll have time to tweak later.

Reply:
Oh, your're right. Let's use `"node", klog.KRef("", nodeName)`. In text format there is no difference, but in Json output will differ. 
```suggestion
				klog.ErrorS(err, "Failed to get node when trying to set owner ref to the node lease", "node", klog.KRef("", nodeName))
```

Reply:
updated to use `klog.KRef("", nodeName))`